eslint.config.mjs

```diff
@@ -178,6 +178,15 @@ export default defineConfig(
           additionalHooks: '(useEnhancedEffect|useIsoLayoutEffect|useEffectAfterFirstRender)',
         },
       ],
+
+      // TODO enable:
+      'react-hooks/refs': 'off',
```

Comment:
Should we add those package per package like we did for the React Compiler ?
Or do you think it's feasible to migrate the whole codebase at once for some of those rules ?

Reply:
We should tackle the per package, but we're still figuring out if we should enable react compiler at all. And if we do, perhaps it should only be ab opt-in per component.

For now I just added these to make eslint shut up after a recent update to react-hooks in code-infra